src/ResourceManagement/Insights/Microsoft.Azure.Insights/Generated/Insights/MetricDefinitionsOperations.cs

```diff
@@ -96,7 +97,6 @@ internal MetricDefinitionsOperations(InsightsClient client)
             var _baseUrl = this.Client.BaseUri.AbsoluteUri;
             var _url = new System.Uri(new System.Uri(_baseUrl + (_baseUrl.EndsWith("/") ? "" : "/")), "{resourceUri}/providers/microsoft.insights/metricDefinitions").ToString();
             _url = _url.Replace("{resourceUri}", resourceUri);
-            _url = _url.Replace("{subscriptionId}", System.Uri.EscapeDataString(this.Client.SubscriptionId));
```

Comment:
That was an error in the spec that Amar pointed out yesterday: the subid was not in the uri of the spec, but it was in the parameter list. 
The replace function would have no effect since there is no {subscriptionId} in _url. There is no harm in removing it.